src/assets/icons/shuffle-random-svgrepo-com.svg

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
@hoangmirs I think we don't need this file, do we?

Reply:
Good catch, removed in https://github.com/nimblehq/til-web/pull/50/commits/1b2c461245697b20450d76da467ae8cec392d7a8